src/content/docs/en/guides/assets.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Assets (Experimental)
+description: Learn how to enable experimental asset support in Astro.
+i18nReady: false
+---
+
+**Built-in Asset Support** is enabled in Astro behind an experimental flag. This built-in feature is intended to replace the optional `@astrojs/image` integration. The new assets experience features:
+
+- Brand new, more intuitive `Image` component
+- Automatic image optimization in Markdown and MDX files
+- Integration with [Content Collections](/en/guides/content-collections/)
+- Improved error messages and types
+
+:::caution
+Assets are an _experimental_ Astro feature. Be mindful that the API is subject to change before it is marked as stable.
+:::
+
+## **Enabling Assets in Your Project**
+
+To enable built-in asset support, you must add the following lines to your `astro.config.mjs` configuration file.
+
+```diff
+ // astro.config.mjs
+ import { defineConfig } from 'astro/config';
+
+ export default defineConfig({
++ experimental: {
++   assets: true
++ }
+ });
+```
+
+## New signature for images imports
+
+Previously, importing an image would return a simple `string` with the path of the image. With the new `image` features enabled, imported image assets now match the following signature:
+
+```ts
+interface ImageAsset {
+  src: string;
+  width: number;
+  height: number;
+  format: string;
+}
+```
+
+## New `src/assets` directory
+
+We recommend putting images in the `src/assets` directory. This is entirely optional, but will become the recommended location for other asset types in the future.
+
+Your images stored in `src/assets/` can be used by components (`.astro`, `.mdx` and other UI frameworks) and in Markdown files.
+
+Import them from a **relative file path** or [import alias](/en/guides/aliases/) in any component file and then use the import as the `Image` component's `src` attribute.
+
+```astro
+---
+// src/pages/index.astro
+import { Image } from 'astro:assets';
+
+// Access images in `src/images/`
+import logo from '../assets/images/logo.png';
+---
+<Image src={logo} alt="Astro" />
+```
+
+## New Markdown support
+
+Relative images can now be referenced in Markdown and MDX files. These will automatically be optimized and hashed by Astro's build process.
+
+Remote images are also valid, but are not automatically optimized by Astro's build process.
+
+```md
+<!-- src/pages/post-1.md -->
+# My Markdown Page
+
+<!-- Local image stored at src/assets/stars.png -->
+![A starry night sky.](../../assets/stars.png)
+<img src="/_assets/stars.hash.png" alt="A starry night sky.">
```

Comment:
Why does this `src` value work? Is `/_assets` a special path that gets mapped to `src/assets`? Would `../../assets/` work?

Reply:
`/_assets` is the wrong path, Nate probably meant `/_astro` (which is the built path for assets in SSG)

Reply:
Whoops I did indeed.

Reply:
Updated to `/_astro`.

To @Jutanium's original question, this `src` value works because the image integration now automatically processes images in Markdown.